README.md

```diff
@@ -36,13 +38,13 @@ A deployed instance of the ReactUSWDS Storybook is located at: [https://trusswor
 Install this package with npm or yarn:
 
 ```
-yarn add @trussworks/react-uswds
+yarn add @trussworks/react-uswds focus-trap-react@^10.2.3
 ```
 
 or
 
 ```
-npm i @trussworks/react-uswds
+npm i @trussworks/react-uswds focus-trap-react@^10.2.3
```

Comment:
Since this is already listed as a `peerDependency`, why should we also tell folks to add it manually here? Wouldn't this have the same effect anyway?

Reply:
Makes sense — for what it's worth, peerDeps worked fine in our project. So i think it might only affect those running older yarn/npm installs or if they notice some particular issue. 

However, this is true of all our peerDeps:
https://github.com/trussworks/react-uswds/blob/ae135939e00b77acecbf64ea46117868e78bfa42/package.json#L71-L76

So in order to avoid a very long top-level instruction here to install all peerDeps manually, what do you think about keeping this as the original and adding a note immediately below (maybe a link to FAQs if it's long) about manually installing any/all peerDeps if they're running an older yarn/npm install or if they run into a particular issue?

Reply:
I reverted the lines for npm/yarn and added a section immediately after install about peer dependencies

Reply:
Great — though since we auto-update those packages, I'd worry about that getting out of sync with package.json.  Maybe we link to it like "This library relies on [some peer dependencies](./package.json). Please verify..."

Reply:
Done.